src/index.html

```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="searchBar"
+      action=""
+      method="post"
+    >
+      <div class="inputWraper inputWraperBig" data-qa="big">
+        <label for="searchBig">
+          <input class="input"
+            data-qa="keypress"
+            type="search"
+            name="searchBig"
+            placeholder="Try &#34Los Angeles&#34"
+            id="searchBig"
+          >
+        </label>
+        <img
+          class="icon"
+          src="./images/Search.svg"
+          alt="search"
+        >
```

Comment:
or you can add this picture as background-image

Reply:
Thank you! I've updated.